pkg/install/hypershift.go

```diff
@@ -299,6 +300,16 @@ func (c *UpgradeController) runHypershiftInstall(ctx context.Context, controller
 	}
 	args = append(args, telemetryArgs...)
 
+	// Enable RHOBS
+	if strings.EqualFold(os.Getenv("ENABLE_RHOBS_MONITORING"), "true") {
+		c.log.Info("ENABLE_RHOBS_MONITORING=true, adding --rhobs-monitoring=true")
+		rhobsArgs := []string{
+			"--rhobs-monitoring",
+			"true",
+		}
+		args = append(args, rhobsArgs...)
+	}
+
```

Comment:
we also need a `ENABLE_RHOBS_MONITORING=1` env var in the job that runs the install. Otherwise, our install fails:
https://github.com/openshift/hypershift/blob/a4fbc63f386561932e07253181d0fce788eb5ba4/cmd/install/install.go#L124-L126
The reason is that during init, we need to know which groupVersion to add to the schema:
https://github.com/openshift/hypershift/blob/a4fbc63f386561932e07253181d0fce788eb5ba4/api/scheme.go#L63-L67

Reply:
@csrwng Can you please review again?